meetings/2025-02-26.md

```diff
@@ -28,16 +32,73 @@
+* JW: With Ruben I will only clarify some details and possible misunderstanding. It would be good if someone else would go throug the specification. If it is there we can create PR in with suggested note. If it is not there, we need to procede with the current PR. There is a question of raising a note to say that SPARQL Update may be reintroduced. In both cases we should cc PAC. 
+* eP: I believe SPARQL update is not in the spec. Since Ruben was working on N3 patch, I assume he's fully aware of the issue Maxime is raising. I assume Ruben should have a good recollection of the reason for dropping SPARQL update.
+* JW: The reason to drop SPARQL update was related to HTTP responses, even if you have append only access. Which would leak information. Custom response codes could be use with N3 Patch. We could also use SPARQL Update with 
+* eP: woudl it be possible to use SPARQL update with a different/customized media type.
+* JW: sounds like a good idea, bring it up.
+* ML: I looked into the source code of CSS, it is only 3 lines that need to be changed to allow blank nodes. See https://github.com/CommunitySolidServer/CommunitySolidServer/blob/e5e7b95327f18b032742d8414e0529913a6ba4de/src/http/input/body/N3PatchBodyParser.ts#L124
+* HZ: I think we have a path forward.
+
 ### Group Based Access Policies
 
+* eP: Simple diagram with Organizations, Roles/Teams, and People
+
+![image](https://hackmd.io/_uploads/SkcouAockl.png)
+
+* HZ: It was discussed during last Solid Practicioners meeting. My opinion changes every time, there are issues related to an information leak. I think we agreed to have a separate STM. I remember that Sebastian from ActivityPods
+* eP: if the authorization system does not have access to the organization specific teams/groups, it cannot efficiently enforce authorization. I think we should differentiate membership from autonomous organization from roles within an org.
+* eP: Yesterday also Christoph preesnted their work https://publikationen.bibliothek.kit.edu/1000174700/154929747
+* HZ: I think proxies can be a good think, it breaks the audit trail. 
+* Michal: I have another issue with the gruop. When the vcard group is colocated with the resources, then it doesn't need to be public. This would mean that the list would be distributed and copied. I also have other issues like scalability.
+* JW: Delegated access, it is worth to be aware that OIDC4VP is quite popular and UMA flow is used in current Solid OIDC standards. I would expect that within LWS it is quite likely that a lot of delegation will be done using OIDC4VP flow.
+* https://openid.net/specs/openid-connect-4-verifiable-presentations-1_0-07.html
+* Michal: I'm aware of VCs and how they can resolve issue with groups. It always depends on some issuer and it seems to be a centralized authority. It may also be an issue for the kind of groups Pavlik mentioned. 
```

Comment:
```suggestion
* JW: Delegated access, it is worth to be aware that OIDC4VP is quite popular and UMA flow is used in current Solid OIDC standards. I would expect that within LWS, it is quite likely that a lot of delegation will be done using OIDC4VP flow.
* https://openid.net/specs/openid-connect-4-verifiable-presentations-1_0-07.html
* Michal: I'm aware of VCs and how they can resolve issue with groups. It always depends on some issuer, and it seems to be a centralized authority. It may also be an issue for the kinds of group Pavlik mentioned. 
```